apps/android/README.md

```diff
@@ -17,12 +17,22 @@ yarn
 yarn build
 ```
 
-2. Then go into `apps/android` folder and simply run the the following
+2. Start the server by running:
 
 ```sh
+cd apps/android
+yarn start
+```
+
+3. Open a new command prompt and run:
+
+```sh
+cd apps/android
 yarn android
 ```
 
+to boot the app. You may need to boot the emulator before running this step.
```

Comment:
You shouldn't have to manually boot the emulator. The CLI should take care of it.

```suggestion
to boot the app.
```

Reply:
Unfortunately the emulator booting doesn't work for me, so I usually have to manually boot it for the script to work.

Reply:
Do you get an error when it tries to boot the emulator? Would be great to figure out why it doesn't work for you.

Reply:
error Failed to launch emulator. Reason: No emulators found as an output of `emulator -list-avds`.
warn Please launch an emulator manually or connect a device. Otherwise app may fail to launch.